### PR TITLE
Added weatherapp icon and title

### PR DESCRIPTION
**Pull request by Allyeeah**
Added weatherapp icon and title. Also modify the merge conflicts.

### CODE REVIEW

File: Weatherapp/app/src/main/java/com/example/weather_app/MainActivity.kt [hunk 1]
```diff
@@ -22,6 +23,36 @@ import androidx.compose.ui.draw.shadow
...
+import com.example.weather_app.models.Location
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.items
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.painterResource
+>>>>>>> SAMPLES/main
```
Comment (officialgio) [inline Weatherapp/app/src/main/java/com/example/weather_app/MainActivity.kt]: Fix the `>>>>>>>` from the merge conflicts. Go over each file you modify and ensure they're not there.

File: Weatherapp/app/src/main/java/com/example/weather_app/MainActivity.kt [hunk 2]
```diff
@@ -38,11 +69,45 @@ class MainActivity : ComponentActivity() {
         // Setting up the UI
         setContent {
             WeatherappTheme {
+                // Call the weather display composable
+                //WeatherDisplay(location = "Chicago") // Replace with your desired location
```
Comment (officialgio) [inline Weatherapp/app/src/main/java/com/example/weather_app/MainActivity.kt]: Do you need the commented code? if not, please remove it.

File: Weatherapp/app/src/main/java/com/example/weather_app/MainActivity.kt [hunk 3]
```diff
@@ -121,11 +187,26 @@ class MainActivity : ComponentActivity() {
...
+=======
+        if (loading) {
+            CircularProgressIndicator()
+        } else if (errorMessage != null) {
+            // Display error message
+            Text(text = "Error: $errorMessage")
+        } else if (weatherData != null) {
+            // Display weather data
+            WeatherCard(weather = weatherData!!)
+            //WeatherInfo(weather = weatherData!!)
```
Comment (officialgio) [inline Weatherapp/app/src/main/java/com/example/weather_app/MainActivity.kt]: Do you need this?

File: Weatherapp/app/src/main/java/com/example/weather_app/MainActivity.kt [hunk 4]
```diff
@@ -138,11 +219,22 @@ class MainActivity : ComponentActivity() {
             "${weather.location.name}, South Korea"
         } else {
             "${weather.location.name}, ${weather.location.country}"
+=======
+        // TODO: Card implementation
+        Log.d("WeatherCard", "Displaying weather for: ${weather.location.name}") // add log
```
Comment (officialgio) [inline Weatherapp/app/src/main/java/com/example/weather_app/MainActivity.kt]: Remove `// add log` this is too obvious.

File: Weatherapp/app/src/main/java/com/example/weather_app/MainActivity.kt [hunk 5]
```diff
@@ -138,11 +219,22 @@ class MainActivity : ComponentActivity() {
             "${weather.location.name}, South Korea"
         } else {
             "${weather.location.name}, ${weather.location.country}"
+=======
+        // TODO: Card implementation
```
Comment (officialgio) [inline Weatherapp/app/src/main/java/com/example/weather_app/MainActivity.kt]: Is this a TODO? Did you alraedy implement the TODO? If so, remove the `TODO` comment.

File: Weatherapp/app/src/main/java/com/example/weather_app/MainActivity.kt [hunk 6]
```diff
@@ -197,12 +289,39 @@ class MainActivity : ComponentActivity() {
...
+=======
+                .padding(16.dp),
+            elevation = CardDefaults.cardElevation(defaultElevation = 10.dp),
+            shape = RoundedCornerShape(16.dp),
+            colors = CardDefaults.cardColors(containerColor = MaterialTheme.colorScheme.primaryContainer)
+        ) {
+            Column(
+                modifier = Modifier.padding(16.dp)
+            ) {
+                // add image
```
Comment (officialgio) [inline Weatherapp/app/src/main/java/com/example/weather_app/MainActivity.kt]: Remove `// add image`.

File: Weatherapp/app/src/main/java/com/example/weather_app/MainActivity.kt (hunk 1 above)
Comment (Allyeeah) [reply Weatherapp/app/src/main/java/com/example/weather_app/MainActivity.kt]: Resolved

Comment (Allyeeah) [reply Weatherapp/app/src/main/java/com/example/weather_app/MainActivity.kt:55]: Resolved

File: Weatherapp/app/src/main/java/com/example/weather_app/MainActivity.kt (hunk 2 above)
Comment (Allyeeah) [reply Weatherapp/app/src/main/java/com/example/weather_app/MainActivity.kt]: Resolved

File: Weatherapp/app/src/main/java/com/example/weather_app/MainActivity.kt (hunk 3 above)
Comment (Allyeeah) [reply Weatherapp/app/src/main/java/com/example/weather_app/MainActivity.kt]: Resolved

File: Weatherapp/app/src/main/java/com/example/weather_app/MainActivity.kt (hunk 4 above)
Comment (Allyeeah) [reply Weatherapp/app/src/main/java/com/example/weather_app/MainActivity.kt]: Resolved

File: Weatherapp/app/src/main/java/com/example/weather_app/MainActivity.kt (hunk 5 above)
Comment (Allyeeah) [reply Weatherapp/app/src/main/java/com/example/weather_app/MainActivity.kt]: Resolved

File: Weatherapp/app/src/main/java/com/example/weather_app/MainActivity.kt (hunk 6 above)
Comment (Allyeeah) [reply Weatherapp/app/src/main/java/com/example/weather_app/MainActivity.kt]: Resolved

**Pull request merged**